src/__init__.py

```diff
@@ -1,34 +1,52 @@
 import os
+import json
 
 from flask import Flask
 from flask_migrate import Migrate
 from dotenv import load_dotenv
 
 from src.models import db
-from src.user import usr
-from src.config.config import TestingConfig
+from src.google import google_bp
+from src.manage import create_tables
 
 load_dotenv()
 
+def get_db_url():
+    # Heroku hack
+    db_url = str(os.environ.get('DATABASE_URL'))
+    if db_url.startswith('postgres://'):
+        db_url = db_url.replace('postgres://', 'postgresql://',1)
+    return db_url
+
+def get_secret():
+    SECRET_KEY = os.environ.get('SECRET_KEY')
+    if not SECRET_KEY:
+        return 'S9M3_SeCR3T'
+    return SECRET_KEY
+    
+
 def create_app(test_config=None):
     app: Flask = Flask(__name__, instance_relative_config=True)
     
-    if test_config is None:
+    if  not test_config:
         app.config.from_mapping(
-            SECRET_KEY=os.environ.get('SECRET_KEY'),
-            SQLALCHEMY_DATABASE_URI=os.environ.get('SQLALCHEMY_DATABASE_URI'),
+            SECRET_KEY=get_secret(),
```

Comment:
Instead of get_secret(), just do this before line 31.

secret = os.environ.get('SECRET_KEY')
if not secret:
    return an error or throw an error.

That's all I have been trying to explain.